lib/main.dart

```diff
@@ -111,15 +113,35 @@ class _PerkPageState extends State<PerkPage> {
                     print('${returnedList[0].isEnabled}');
                   },
                 ),
+              ],
+            ),
+            Expanded(
```

Comment:
Personally, I think that this would be better as a `ListTile` inside of the `ListView` above, with a trailing `Switch`.
It's also slightly nicer UX if you move the setState call to a `ListTile.onTap` method, e.g.
```suggestion
            ListTile(
                title: Text('Dark Theme'),
                trailing: Switch(
                    value: lightModeDisabled,
                    onChanged: (value) {} // this is a required parameter and is needed for the switch to change colour when toggled
                ),
                onTap: () => setState(() => lightModeDisabled = !lightModeDisabled)
            )
```